pkg/apis/crd/v1alpha1/types.go

```diff
@@ -434,6 +435,11 @@ type NetworkPolicyPeer struct {
 	//  Exact FQDNs, i.e. "google.com", "db-svc.default.svc.cluster.local"
 	//  Wildcard expressions, i.e. "*wayfair.com".
 	FQDN string `json:"fqdn,omitempty"`
+	// Select certain Nodes which match the label selector,
+	// if no nodeSelector is specified, then all Nodes will be selected
+	// in the cluster.
```

Comment:
I don't think this behavior is expected. It basically breaks compaibility of existing rules.

Reply:
Updated.